assets/dev/scss/frontend/general/_button.scss

```
@@ -25,16 +25,24 @@
 	}
 
 	&-icon {
+
+		i {
+			color: var(--button-icon-color, inherit);
+		    font-size: var(--button-icon-size);
+		}
+		
 		svg {
-			width: 1em;
 			height: auto;
+			width: var(--button-icon-size, 1em);
+			fill: var(--button-icon-color, inherit);
```

Maybe we should introduce this CSS variable for the title/text: `--button-icon-color`.

```suggestion
			fill: var(--button-icon-color, --button-title-color);
```

---

My IDE is giving me an error about this 'inherit' styling value.
Can we try to get rid of it?

---

where is the value for that variable supposed to come?